tests/functional/test_accounts.py

```diff
@@ -47,6 +47,18 @@ def test_sign_message(signer):
     assert signer.check_signature(message, signature)
 
 
+def test_sign_string(signer):
+    message = "Hello Apes!"
+    signature = signer.sign_message(message)
+    assert signer.check_signature(message, signature)
+
+
+def test_sign_message_unsupported_type_returns_none(signer):
+    message = 1234
```

Comment:
Let's use a float or something weirder. I think `int` should be valid personally, just convert to hex first.

```
int_to_str = HexBytes(my_int).hex()
```

Reply:
got it, added support for signing/verifying ints directly, and updated this test case to pass a float